hs_modelinstance/models.py

```diff
@@ -7,7 +7,10 @@
 from mezzanine.core.models import Ownable
 from mezzanine.pages.page_processors import processor_for
 from hs_core.models import AbstractResource, resource_processor, CoreMetaData, AbstractMetaDataElement
-
+from django.shortcuts import get_object_or_404
+from django.contrib.sites.models import get_current_site
+from hs_model_program.models import ModelProgramResource
+from hs_core.signals import *
```

Comment:
If you are going to add imports, please keep the order neat.

- Standard Library Modules
- Django Modules
- Mezzanine Modules
- 3rd Party Modules
- Hydroshare Modules